src/components/PrefetchBalancesWrapper/PrefetchBalancesWrapper.tsx

```diff
@@ -30,7 +30,8 @@ const hasUnfetchedBalancesAtom = atom<boolean>(true)
 export default function PrefetchBalancesWrapper({
   children,
   shouldFetchOnAccountUpdate,
-}: PropsWithChildren<{ shouldFetchOnAccountUpdate: boolean }>) {
+  className,
+}: PropsWithChildren<{ shouldFetchOnAccountUpdate: boolean; className?: string }>) {
```

Comment:
Why is classname needed?

Reply:
this lets us make a styled-components wrapper in the other file

Reply:
`className` is what styled components use to target changes, so if you don't pass it through then you can't styled-componentize the component (eg `const StyledWrapper = styled(PrefetchBalancesWrapper)`)